src/util/screening-selector.js

```diff
@@ -17,7 +18,9 @@ export function getScreeningInstrument() {
     return [questionnaireWhoAudit, elmJsonWhoAudit, valueSetJsonUsAudit];
   } else if (screeningInstrument == 'nidaqs2usaudit') {
     return [questionnaireNidaQs, elmJsonNidaQs, valueSetJsonUsAudit];
+  } else if (screeningInstrument == 'phq9') {
+    return [questionnairePhq9, elmJsonPhq9, valueSetJsonPhq9];
```

Comment:
@mcjustin I was getting JS runtime errors: `elmJsonPhq9` & `valueSetJsonPhq9` aren't defined.  Since we don't currently have a phq9 cql json or valuset json defined, I think we could pass in empty object for now.  
```suggestion
    return [questionnairePhq9, {}, {}];
```

Reply:
I think I'll take a look at defining those 😁